### PR TITLE
wasm loader: Fix pop invalid offset count when stack top is ANY

### PR DESCRIPTION
**Pull request by wenyongh**
In wasm_loader_pop_frame_offset, when the stack is in polymorphic state
and the stack top operand is VALUE_TYPE_ANY, if we popping I64/F64
operand, we should pop one offset but not two offsets.

The issue was reported in #3513 and #3514.

### CODE REVIEW

File: core/iwasm/interpreter/wasm_loader.c [hunk 1]
```diff
@@ -10888,6 +10887,12 @@ wasm_loader_get_custom_section(WASMModule *module, const char *name,
 }
 #endif
 
+#if 0
+#define HANDLE_OPCODE(opcode) #opcode
+DEFINE_GOTO_TABLE(const char *, op_mnemonics);
```
Comment (TianlongLiang) [inline core/iwasm/interpreter/wasm_loader.c:10892]: Is this mnemonics opcode table aimed to facilitate debugging in wasm_loader?

Comment (wenyongh) [reply core/iwasm/interpreter/wasm_loader.c:10892]: Yes, in wasm_loader_prepare_bytecode, when traversing opcodes, we can print opcode name, e.g. `printf("##%s\n", op_mnemonics[opcode]);`.

Comment (TianlongLiang) [reply core/iwasm/interpreter/wasm_loader.c:10892]: thanks, got it

**Comment by TianlongLiang** [comment]
LGTM

**Pull request merged**